core/spam/engine.go

```diff
@@ -130,6 +130,7 @@ func New(log *logging.Logger, config Config, epochEngine EpochEngine, accounting
 	}
 
 	e.transactionTypeToPolicy[txn.ProposeCommand] = proposalPolicy
+	e.transactionTypeToPolicy[txn.BatchProposeCommand] = proposalPolicy
```

Comment:
Is this correct to split the policies like this? Because now we have two spam policies for proposals, one for batch and one for normal proposals, so I can actually submit `2 * SpamProtectionMaxProposals` per epoch if I split them across the two transaction types.

Reply:
Yeah true. Probably not correct